infra-as-code/bicep/modules/vwanConnectivity/vwanConnectivity.bicep

```diff
@@ -0,0 +1,199 @@
+              Virtual Hub
+              VPN Gateway
+              ExpressRoute Gateway
+              Azure Firewall
+AUTHOR/S: Fai Lai @faister
+VERSION: 1.0.0
+*/
+
+@description('Prefix value which will be prepended to all resource names. Default: alz')
+param parCompanyPrefix string = 'alz'
+
+@description('The IP address range in CIDR notation for the vWAN virtual Hub to use. Default: 10.100.0.0/23')
+param parVhubAddressPrefix string = '10.100.0.0/23'
+
+@description('Switch which allows Azure Firewall deployment to be disabled. Default: false')
+param parAzureFirewallEnabled bool = false
+
+@description('Azure Firewall Tier associated with the Firewall to deploy. Default: Standard ')
+@allowed([
+  'Standard'
+  'Premium'
+])
+param parAzureFirewallTier string = 'Standard'
+
+@description('Public IP Address SKU. Default: Standard')
+@allowed([
+  'Basic'
+  'Standard'
+])
+param parPublicIPSku string = 'Standard'
+
+@description('Tags you would like to be applied to all resources in this module. Default: empty array')
+param parTags object = {}
+
+@description('Switch which allows Virtual Hub. Default: true')
+param parVirtualHubEnabled bool = true
+
+@description('Switch which allows VPN Gateway. Default: false')
+param parVPNGatewayEnabled bool = false
+
+@description('Switch which allows ExpressRoute Gateway. Default: false')
+param parERGatewayEnabled bool = false
+
+@description('Prefix Used for Virtual WAN. Default: {parCompanyPrefix}-vwan-{resourceGroup().location}')
+param parVWanName string = '${parCompanyPrefix}-vwan-${resourceGroup().location}'
+
+@description('Prefix Used for Virtual Hub. Default: {parCompanyPrefix}-hub-{resourceGroup().location}')
+param parVHubName string = '${parCompanyPrefix}-vhub-${resourceGroup().location}'
+
+@description('Prefix Used for VPN Gateway. Default: {parCompanyPrefix}-vpngw-{resourceGroup().location}')
+param parVPNGwName string = '${parCompanyPrefix}-vpngw-${resourceGroup().location}'
+
+@description('Prefix Used for ExpressRoute Gateway. Default: {parCompanyPrefix}-ergw-{resourceGroup().location}')
+param parERGwName string = '${parCompanyPrefix}-ergw-${resourceGroup().location}'
+
+@description('Azure Firewall Name. Default: {parCompanyPrefix}-fw-{resourceGroup().location}')
+param parAzureFirewallName string = '${parCompanyPrefix}-fw-${resourceGroup().location}'
+
+@description('Region in which the resource group was created. Default: {resourceGroup().location}')
+param parLocation string = resourceGroup().location
+
+@description('The scale unit for this VPN Gateway: Default: 1')
+param parVPNGwScaleUnit int = 1
+
+@description('The scale unit for this ExpressRoute Gateway: Default: 1')
+param parERGwScaleUnit int = 1
+
+@description('Set Parameter to true to Opt-out of deployment telemetry')
+param parTelemetryOptOut bool = false
+
+// Customer Usage Attribution Id
+var varCuaid = '7f94f23b-7a59-4a5c-9a8d-2a253a566f61'
```

Comment:
Has this been added to the wiki page table for these: https://github.com/Azure/ALZ-Bicep/wiki/CustomerUsage

Reply:
added